grama/core.py

```diff
@@ -34,6 +34,11 @@
 from toolz import curry
 
 
+
+#######
+from collections import Iterable
```

Comment:
I don't see `Iterable` used in the code here... is this import necessary? If not, please remove.